pkg/images/tasks.go

```diff
@@ -85,7 +85,12 @@ func GetImage(runtime connector.ModuleRuntime, kubeConf *common.KubeConf, name s
 	} else {
 		corednsTag = "1.8.0"
 	}
-
+	multusRepoAddr := kubeConf.Cluster.Registry.PrivateRegistry
+	if multusRepoAddr == "" {
+		// TODO
```

Comment:
The image of multus has been synchronized to repo of kubesphere. 
Maybe `kubesphere/multus-cni:v3.8` can be used directly here.